shared/cfg/guest-hw.cfg

```diff
@@ -28,6 +32,8 @@ variants:
         # following lines to enable the vhost support ( only available
         # for tap )
         #netdev_extra_params = ",vhost=on"
+        q35:
+            nic_pci_bus = pcie_switch
```

Comment:
The only issue with this is, that if the test defined other nic:

```
nics += " test_nic"
nic_model_test_nic = e1000
```
it'd have been added into the `pcie_switch`. I'm not saying it's a problem, we should just check the existing (up/down-stream) configs for this.

An alternative solution would be not to depend on the pci name, but rather pci type. The framework is prepared for it, it can handle `{'type': 'PCI'}` or `PCIE`, if they are defined like that. Then one would only say I have this `nic`, it's `PCIE` and avocado should pick the first free `PCIE` bus. Obviously we'd have to still provide some hints on which one to use (to avoid using the root complex or some switches in between).

Anyway that was just for consideration, I believe right now the hardcoded values serve the best.